decision-making/voting-on-proposals.md

```diff
@@ -0,0 +1,30 @@
+# Decision-making
+
+### Proposal submission
+Proposal is submitted by opening pull request to this repository.
+Initiator must present the idea on the architecture meeting.
+The announcement should include problem description the proposal is trying to resolve as well as solution(s).
+
+### Discussion
+There must be minimum 2 weeks of discussion in the pull request thread.
+After that initiator may decide when to start the vote.
+
+### Voting
+All eligible voters must get notification about the voting start.
+A valid voting period must be announced when voting is started, must be greater that 2 weeks and must not be changed during the vote.
+
+### Approval
```

Comment:
Are we ok with accepting proposals that have 100% opposition from Magento team but enough votes from community to qualify?

Reply:
From my point of view - yes, but I need more thoughts and concerns here.

Reply:
More sophisticated model with someone having the power of veto may be implemented.

Reply:
Don't think that we need to overcomplicate the voting process with veto right. I think every vote should have the same power.
For me, it's hard to imagine a situation when a proposal will get 100% opposition from Magento. It has never happened before, why should be foreseen in the future? 
Even internally for many questions, we don't have a single opinion.

Reply:
*I fully support the initiative of public voting on proposals and will provide further input on design, goals, etc.* I am commenting on this facet separately for sake of expediency.

**A veto mechanism is neither necessary nor appropriate.** Whereas Magento, an Adobe company (MAAC) _alone_ releases the product we all call "Magento", I propose to add a note which recognizes the following reality:

> When a majority vote exists for a proposal, it does not guarantee that the proposal will be adopted or implemented.

This approach allows for two important realties:

1. To commemorate the collective opinion of the balloters in the face of disagreement with Magento/Adobe strategic needs
2. To allow Magento/Adobe to make even-better-informed product decisions based on our business needs.

As an addendum to (2), I believe this will force MAAC to be aware of & to justify product decisions which are in contrast to the wishes of the our internal & external ecosystem members, as proxied by balloters.